### DISCUSSION TITLE
the latest  coplilot unable to install on the latest official package IntelliJ IDEA 2022.1.2, prompting installation error

### DISCUSSION THREAD

**Discussion by oday-xiexin** [community/community#17638] [Copilot Conversations] [answered]
coplilot unable to install on the latest official package IntelliJ IDEA 2022.1.2 (Ultimate Edition), prompting installation error

**Comment by oday-xiexin** [comment]
coplilot version 1.1.24.1610

**Comment by jansorg** [comment, marked as answer]
An older version did not properly terminate the Copilot helper process. Please terminate processes named `copilot-agent-windows` (if you're on Windows) or just restart your machine and IDE as a quick fix.
If your error is something else, please post the error message you're seeing.
